github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
```

Comment:
My 2cents: I like this syntax is better, easier to identify the title.

Also, the number of equals is not relevant
=

It's just a style preference, that happens to be worth the effort for me.